contrib/storage-phoenix/pom.xml

```diff
@@ -29,9 +29,9 @@
   <name>Drill : Contrib : Storage : Phoenix</name>
 
   <properties>
-    <phoenix.version>5.1.2</phoenix.version>
+    <phoenix.version>5.1.3</phoenix.version>
     <!-- Keep the 2.4.2 to reduce dependency conflict -->
```

Comment:
@cgivre I need to reword this comment slightly and prune some commented sections from java-exec/pom.xml below, but are you happy with this PR at this stage?